src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { PostsActionCreators }
+  from '../../redux/reducers/posts/action-creators';
+import { SelectedPost } from '../../redux/reducers/posts/selectors';
+import './NewCommentForm.scss';
+
+export const NewCommentForm: React.FC = () => {
+  const dispatch = useDispatch();
+  const selectedPostId = useSelector(SelectedPost);
+
+  const [nameValue, setNameValue] = useState('');
+  const [emailValue, setEmailValue] = useState('');
+  const [bodyValue, setBodyValue] = useState('');
+
+  // eslint-disable-next-line
+  const addNewCommentHandler = (e: any) => {
```

Comment:
Don't use `any` there are needed types that you can import from `'react'` package